service/src/test/resources/WorkPresentationBean/basic/source.json

```diff
@@ -1,26 +1,88 @@
 {
     "title": "fjeldsange",
     "workId": "work-of:870970-basis:00010529",
-    "dbUnits": { "unit:64": [ { "pid": "870970-basis:00010529",
-                "title": "fjeldsange", "types": [ "Node" ],
-                "creators": [ "yrjö kilpinen" ], "subjects": [ ],
-                "fullTitle": "Fjeldsange : for 1 stemme og klaver" }, {
+    "dbUnits": {
+        "unit:64": [
+            {
+                "pid": "870970-basis:00010529",
+                "title": "fjeldsange",
+                "types": [
+                    "Node"
+                ],
+                "creators": [
+                    {
+                        "type": "aut",
+                        "value": "yrjö kilpinen"
+                    }
+                ],
+                "subjects": [ ],
+                "fullTitle": "Fjeldsange : for 1 stemme og klaver"
+            },
+            {
                 "pid": "800010-katalog:99122054413305763",
-                "title": "fjeldsange", "types": [ "Node" ],
-                "creators": [ ], "subjects": [ ],
-                "fullTitle": "Fjeldsange : for 1 stemme og klaver" }, {
-                "pid": "710100-katalog:00010529", "title": "fjeldsange",
-                "types": [ "Node" ], "creators": [ "yrjö kilpinen" ],
+                "title": "fjeldsange",
+                "types": [
+                    "Node"
+                ],
+                "creators": [ ],
```

Comment:
Hvad blev der af den creator her i pid 800010-katalog:99122054413305763 ?

Reply:
Den er tom lige som i den gamle, for at se at man kan håndtere tomme lister.

Reply:
Ahh ja, sad ellers og stirrede mig blind på den og synes der manglede noget. Men du har ret.